src/main/java/com/dorin/learning/task1/Main.java

```diff
@@ -0,0 +1,31 @@
+package com.dorin.learning.task1;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Main {
+    public static void main(String[] args) {
+        // Create a function that receives a list of numbers, and computes the average for just the positive ones
+        List<Number> numbers = new ArrayList<>();
+        numbers.add(-5);
+        numbers.add(8);
+        numbers.add(7);
+        numbers.add(-21);
+
+        Number average = computeAverage(numbers);
+        System.out.println(average);
+    }
+
+    private static Number computeAverage(List<Number> numbers) {
+        List<Number> positiveOnes = numbers.stream()
```

Comment:
You don't need list here, single stream compute

Reply:
Then I need another operation to find out the number of positive numbers in order to compute the average